packages/ui/src/components/VerificationCode/__stories__/Size.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { Template } from './Template.stories'
+
+export const Size = Template.bind({})
+Size.args = {
+  size: 'small',
+}
+
+Size.parameters = {
+  docs: {
+    description: { story: 'use `size` prop to change the size of the input' },
+  },
+}
```

Comment:
Can we get all the sizes in the storybook ? 

<img width="1059" alt="Screenshot 2023-11-28 at 10 44 05" src="https://github.com/scaleway/ultraviolet/assets/15812968/ef88a218-4d6c-44be-ac94-e53086e908e1">
 
That's my last feedback im approving in advance!

Reply:
Yes i add all sizes